doc/source/ray-overview/index.md

```diff
@@ -34,273 +34,13 @@ To use Ray in Java, first add the [ray-api](https://mvnrepository.com/artifact/i
-:type: ref
-:text: Learn more about Ray Train
-:classes: btn-outline-primary btn-block
-```
-`````
-
-`````{dropdown} <img src="images/ray_svg_logo.svg" alt="ray" width="50px"> Tune: Hyperparameter Tuning at Scale
-:animate: fade-in-slide-down
-
-[Tune](../tune/index.rst) is a library for hyperparameter tuning at any scale. 
-With Tune, you can launch a multi-node distributed hyperparameter sweep in less than 10 lines of code. 
-Tune supports any deep learning framework, including PyTorch, TensorFlow, and Keras.
-
-````{note}
-To run this example, you will need to install the following:
-
-```bash
-pip install "ray[tune]"
-```
-````
-
-This example runs a small grid search with an iterative training function.
-
-```{literalinclude} ../../../python/ray/tune/tests/example.py
-:end-before: __quick_start_end__
-:language: python
-:start-after: __quick_start_begin__
-```
-
-If TensorBoard is installed, automatically visualize all trial results:
-
-```bash
-tensorboard --logdir ~/ray_results
-```
-
-```{link-button} ../tune/index
-:type: ref
-:text: Learn more about Ray Tune
-:classes: btn-outline-primary btn-block
-```
-
-`````
-
-
-`````{dropdown} <img src="images/ray_svg_logo.svg" alt="ray" width="50px"> Serve: Scalable Model Serving
-:animate: fade-in-slide-down
-
-[Ray Serve](../serve/index) is a scalable model-serving library built on Ray. 
-
-````{note}
-To run this example, you will need to install the following libraries.
-
-```{code-block} bash
-pip install "ray[serve]" scikit-learn
-```
-````
-This example runs serves a scikit-learn gradient boosting classifier.
-
-```{literalinclude} ../serve/_examples/doc_code/quick_start.py
-:language: python
-:start-after: __serve_example_begin__
-:end-before: __serve_example_end__
-```
-
-As a result you will see `{"result": "versicolor"}`.
-
-```{link-button} ../serve/index
-:type: ref
-:text: Learn more about Ray Serve
-:classes: btn-outline-primary btn-block
-```
-`````
-
-
-`````{dropdown} <img src="images/ray_svg_logo.svg" alt="ray" width="50px"> RLlib: Industry-Grade Reinforcement Learning
-:animate: fade-in-slide-down
-
-[RLlib](../rllib/index.rst) is an industry-grade library for reinforcement learning (RL) built on top of Ray.
-RLlib offers high scalability and unified APIs for a variety of industry- and research applications.
-
-````{note}
-To run this example, you will need to install `rllib` and either `tensorflow` or `pytorch`.
-```bash
-pip install "ray[rllib]" tensorflow  # or torch
-```
-````
-
-```{literalinclude} ../../../rllib/examples/documentation/rllib_on_ray_readme.py
-:end-before: __quick_start_end__
-:language: python
-:start-after: __quick_start_begin__
-```
-
-```{link-button} ../rllib/index
-:type: ref
-:text: Learn more about Ray RLlib
-:classes: btn-outline-primary btn-block
-```
+`````{dropdown} <img src="images/ray_svg_logo.svg" alt="ray" width="50px"> AI Runtime: Parallelizing Functions with Ray Tasks
 
+TODO(rliaw): put together a quickstart
```

Comment:
right, so what would be the logic here? just one big, unified AIR example to get a taste, and then the more fine-grained lib-specific starters go into the other doc, as proposed? Could work!